monitoring/oplog-populator/alerts.yaml

```diff
@@ -36,6 +39,18 @@ groups:
       description: "Oplog populator failed to configure connector"
       summary: "Oplog populator couldn't update kafka connect connector"
 
+  - alert: KafkaConnectReplicasAboveThreshold
```

Comment:
I feel like the name of the alert might be missleading, as the user might think we are talking about the k8s replicas of kafka connect ?

Reply:
I renamed it as `KafkaConnectConnectorsAboveThreshold`, the description also helps understanding, what do you think?